### PR TITLE
added hasnode links

### PR DESCRIPTION
**Pull request by abhijeet-26**
## Check List (Check all the applicable boxes) 

- [ x] My code follows the code style of this project.
- [ x] My change requires changes to the documentation.
- [x ] I have updated the documentation accordingly.
- [ x] All new and existing tests passed.
- [ x] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3746"></a>

### CODE REVIEW

File: data/abhijeet-26.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  "name": "Abhijeet Basfore",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/abhijeet-26.png",
+  "tags": ["javascript", "Springboot", "DataStructures and Algorithms", "react.js", "Microservices","junit/Mockito","Open source"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/AbhijeetBasfore" },
+    { "icon": "FaGithub", "url": "https://github.com/abhijeet-26" },
+    { "icon": "SiHashnode", "url":"https://hashnode.com/@abhijeet26"},
```
Comment (Aadarsh805) [inline data/abhijeet-26.json:11]: this comma at the end of line 11 needs to be removed

File: data/abhijeet-26.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AbhijeetBasfore",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "LinkedIn: Let's connect",
+      "url": "https://www.linkedin.com/in/abhijeet-basfore-8526b71b5/",
```
Comment (Aadarsh805) [inline data/abhijeet-26.json:26]: There's something wrong with the LinkedIn URL, it takes me to 404 page

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/abhijeet-26.json (hunk 2 above)
Comment (abhijeet-26) [reply data/abhijeet-26.json]: please check now

**Review by Aadarsh805 — APPROVED** [review]
All good  👍

**Review by SaraJaoude — APPROVED** [review]
looks good

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
please resolve conflicts - it looks like your fork may be out of date which is causing conflicts with yourself

here is a link on how to edit a profile from our docs: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Comment by abhijeet-26** [comment]
Okay thank you

**Pull request closed without merging**